tools/rpi-otp-private-key

```diff
@@ -6,6 +6,8 @@ FORCE=0
 READ_KEY=""
 WRITE_KEY=""
 OUTPUT_BINARY=0
+LENGTH=8
```

Comment:
Perhaps LENGTH_IN_ROWS, to be clear this isn't a 'byte' or 'cell' count?

Reply:
I guess that applies to OFFSET as well

Reply:
ROW_COUNT and ROW_OFFSET?

Reply:
ROW would be nice because we don't support non-row aligned updates